docs/calicoctl.md

```diff
@@ -0,0 +1,62 @@
+# calicoctl command line interface user guide
```

Comment:
I think we should call these "reference", not "guide".

Fairly standard terminology in technical docs that "reference" docs are meant for looking up specific things.